src/components/Payroll/PayrollHistory/PayrollHistoryPresentation.tsx

```diff
@@ -221,35 +204,43 @@ export const PayrollHistoryPresentation = ({
         columns={[
           {
             title: t('columns.payPeriod'),
```

Comment:
Some string concat / formatting was happening in the non presentational component. I've moved it here for separation of concerns.